user/pages/03.commerce2/02.developer-guide/12.recurring/01.subscriptions-overview/docs.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Technical overview
+taxonomy:
+    category: docs
+---
+
+This section of the documentation provides an overview of the Subscription entity and related entities, plugins, and objects. Specifically, this section describes:
+* Billing schedules, the billing schedule plugin, Billing periods, and the Prorater plugin
+* The Subscription type plugin and Charges
+* Recurring order types and order item types
+* The PurchasableEntitySubscriptionTrait and the Initial order
+
+## Billing Schedules
+Each subscription has a billing schedule. Billing schedules are configuration entities that are created through the admin UI: `Commerce >> Configuration >> Payment`. Each billing schedule has a type, which is either prepaid or postpaid (charge at the beginning or at the end of the billing period). 
+
+### Billing Schedule plugin
+A billing schedule also has a billing schedule plugin. Two billing schedule plugins are included in the Commerce Recurring module: Rolling and Fixed. Both rolling and fixed billing schedules have an Interval (some number of hours, days, weeks, months, or years). Fixed billing schedules with monthly intervals also have a Start day (1 - 31). Fixed billing schedules with yearly intervals also have a Start month (Jaunuary - December). 
```

Comment:
The first paragraph needs to be rewritten/moved. We should not go into what settings are available before explaining what the plugin is, and what it's used for.

A billing schedule is powered by a billing schedule plugin, which is responsible for generating the billing period. Out of the box Recurring supports rolling and fixed billing periods. Rolling periods starts at the moment of subscription and last until the configured interval passes. Fixed periods start at the beginning of the configured interval (month/year/etc). Then expand from there.